mne/datasets/utils.py

```diff
@@ -143,6 +145,7 @@ def _data_path(path=None, force_update=False, update_path=True, download=True,
         sample='MNE-sample-data-processed.tar.gz',
         spm='MNE-spm-face.tar.bz2',
         somato='MNE-somato-data.tar.gz',
+        brainstorm='sample_ctf',
```

Comment:
this will be changed. I have this just because it's the smallest file to test and I'm on a slow connection